xclim/data/fr.json

```
@@ -201,6 +201,13 @@
     "comment": "",
     "abstract": "Précipitation accumulée totale. Si la température journalière moyenne est donnée, le paramètre phase peut-être utilisé pour restreindre le calcul aux précipitations d'une seule phase (liquide ou solide). Les précipitations sont considérées solides si la température journalière moyenne est sous 0°C (et vice-versa)."
   },
+  "WET_PRCPTOT": {
+    "long_name": "Précipitation totale lors de jours pluvieux",
+    "description": "Précipitation totale {freq:f} lorsque la précipitation journalière excède {thresh}.",
+    "title": "Précipitation accumulée totale lors de jours pluvieux",
+    "comment": "",
+    "abstract": "Précipitation accumulée totale lors de jours pluvieux. Un jour est considéré pluvieux si la précipitation est supérieure ou égale à {thresh}."
```

Can't have format strings in "abstract" or "title".
```suggestion
    "abstract": "Précipitation accumulée totale lors de jours pluvieux. Un jour est considéré pluvieux si la précipitation est supérieure ou égale à un seuil donné."
```